java/system-test/system-test-data-generation/src/main/java/sleeper/systemtest/datageneration/ECSSystemTestTask.java

```diff
@@ -123,12 +122,12 @@ public void run() {
     private static class CommandLineFactory {
         private final AWSSecurityTokenService stsClientV1;
         private final StsClient stsClientV2;
-        private final AmazonSQS sqsClient;
+        private final SqsClient sqsClientV2;
```

Comment:
It looks like we can call the SqsClient just sqsClient rather than v2, since there's no v1 client here.

Reply:
This one hasn't been updated. It's not a big deal though, it'll probably get renamed when this module is updated anyway.